### PR TITLE
[OSD-10352] - Webhook receiver test coverage

### PR DESCRIPTION
**Pull request by T0MASD**
### What type of PR is this?
_(test)_

### What this PR does / why we need it?

To improve Webhook receiver test coverage

### Which Jira/Github issue(s) this PR fixes?

_Fixes #_ [OSD-10352](https://issues.redhat.com/browse/OSD-10352)

### Pre-checks (if applicable):
- [ ] Ran `make test`

### CODE REVIEW

File: pkg/handlers/webhookreceiver.go [hunk 1]
```diff
@@ -268,34 +268,36 @@ func (h *WebhookReceiverHandler) updateNotificationStatus(n *oav1alpha1.Notifica
...
-				resolvedConditionTime := status.Conditions.GetCondition(oav1alpha1.ConditionAlertResolved).LastTransitionTime
-				if firing {
-					// Status transition is Firing to Firing
-					// Do not update the condition for AlertFiring and AlertResolved
-					// Only update the timestamp for the ServiceLogSent
-					status.SetStatus(oav1alpha1.ConditionAlertFiring, "Alert is firing", corev1.ConditionTrue, firedConditionTime)
-					status.SetStatus(oav1alpha1.ConditionAlertResolved, "Alert resolved", corev1.ConditionFalse, resolvedConditionTime)
-					status.SetStatus(oav1alpha1.ConditionServiceLogSent, "Service log sent", corev1.ConditionTrue, timeNow)
+			firingCondition := status.Conditions.GetCondition(oav1alpha1.ConditionAlertFiring)
+			if firingCondition != nil {
```
Comment (mrbarge) [inline pkg/handlers/webhookreceiver.go:272]: This refactoring has no `else`. There should never be a state where we don't update status at all.

Comment (T0MASD) [reply pkg/handlers/webhookreceiver.go:272]: Thanks I've added the else block as suggested

File: pkg/handlers/webhookreceiver_test.go [hunk 2]
```diff
@@ -25,16 +28,13 @@ func (fn RoundTripperFunc) RoundTrip(r *http.Request) (*http.Response, error) {
...
-	dummyJWT = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhbGciOiJIUzI1NiIsInR5cCI6IkJlYXJlciJ9.0rLPJ-zaY_wFsADkvKmW5nsZzeyFmCP0276XSrkctb4"
-)
-
 var _ = Describe("Webhook Handlers", func() {
 
 	var (
 		mockCtrl               *gomock.Controller
 		mockClient             *clientmocks.MockClient
-		testConn               *sdk.Connection
+		mockUpdater            *clientmocks.MockStatusWriter
```
Comment (bmeng) [inline pkg/handlers/webhookreceiver_test.go:36]: not big deal, but it would be better if this names `mockStatusWriter` or `mockStatusUpdater`

**Pull request merged**